custom/icds_reports/tests/test_export_data.py

```diff
@@ -2080,6 +2081,22 @@ def test_beneficiary_export(self):
                             "Generated at",
                             "16:21:11 15 November 2017"
                         ],
+                        [
+                            'State',
+                            'st1'
+                        ],
+                        [
+                            'District',
+                            'd1'
+                        ],
+                        [
+                            'Block',
+                            'b1'
+                        ],
+                        [
+                            'Supevisor',
+                            's1'
+                        ],
```

Comment:
this was the only non-obvious change. now that the AWC has a proper parent line these all get included here. I assume that's the correct behavior, and the previous test case was kind of wonky?